src/components/Input/Input.props.ts

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import { ChangeEventHandler } from 'react'
+
+import { HTMLType, Type } from './Input.types.ts'
+import { IconComponent } from '../Icon/Icon.props.ts'
+
+export type InputProps = {
+
+  /**
+   * The type of the Input.
+   */
+  type?: Type
+
+  /**
+   * The html type of Input.
+   */
+  htmlType?: HTMLType
+
+  /**
+   * Whether the input is disabled.
+   */
+  isDisabled?: boolean,
+
+  /**
+   * Whether the input has error.
+   */
+  isError?: boolean
```

Comment:
```suggestion
  hasErrors?: boolean
```

I would name it like this, more self explanatory